### ISSUE TITLE
Seitennummer 1 erst bei Kapitel 1 anfangen lassen, davor römische Ziffern

### ISSUE THREAD

**Issue by programming-wolf** [isaqb-org/advanced-template#164]
- Kapitel 1 sollte die Seitennummer 1 haben, davor sollten es (front-matter) römische Ziffern sein. 
- Kleines Detail für die Druckaufbereitung als PDF:  Die Seite mit der Nummer 1 liegt momentan bei doppelseitigem Ausdruck links anstatt rechts. Wenn man sie umnummeriert, so dass sie die Nummer iv trägt, dann passt alles, weil aus der Seite 2 dann die Seite 1 wird.

**Comment by alexanderschramm1992** [comment]
Ich habe mir das mal genauer angeschaut. Ich denke wir müssen uns entscheiden, ob wir:

1. entweder Seitennummer 1 erst bei Kapitel 1 anfangen lassen wollen (siehe Titel des Issues)
2. oder der Copyright Seite in der Vorlage die Seitennummer iv geben (siehe Kommentar)

Spätestens wenn aus der Vorlage ein richtiges Curriculum gemacht wird, besteht die Chance, dass die zweite Lösung nicht mehr trägt. Denn dann ist das Inhaltsverzeichnis vielleicht nicht mehr genau eine Seite lang.

Zusatzfrage zu Lösungsoption 1: Ist Kapitel 1 wirklich das erste Inhaltskapitel, oder das erste Element im Inhaltsverzeichnis?

**Comment by programming-wolf** [comment]
Ja, das geht nicht wirklich. im PDF kommt das Copyright immer nach dem TOC, weil die Position da nicht geändert werden kann. Im HTML kommt sie bzwp. zuerst, ist eigentlich noch kein Content. Im PDF ist das halt schon content. 

Evtl. gibt's was in Asciidoc, wo man sagen kann, dass das noch nicht content ist. Wenn nicht, lassen wir es und stellen es auf wontfix.
